src/openassetio-core-c/include/openassetio/c/SimpleMap.h

```diff
@@ -0,0 +1,138 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2013-2022 The Foundry Visionmongers Ltd
+#pragma once
+
+#include <cstddef>
+#include <cstdint>
+
+#include <openassetio/c/export.h>
+
+#include "./StringView.h"
+#include "./namespace.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+/**
+ * @addtogroup CAPI C API
+ * @{
+ */
+
+/**
+ * Opaque handle type representing a @fqref{SimpleMap} "SimpleMap"
+ * instance.
+ *
+ * The associated @fqcref{SimpleMap_s} "SimpleMap suite" functions
+ * operate on this handle type.
+ */
+// NOLINTNEXTLINE(modernize-use-using)
+typedef struct OPENASSETIO_NS(SimpleMap_t) * OPENASSETIO_NS(SimpleMap_h);
```

Comment:
Do we want the _h from SimpleMap_h, if it's the type that should be most used?
Feels a bit odd to have an handle type that we don't much care what the type is meant to be, but then have everyone type '_h' everywhere so we're reminded it's a handle! :smile:

Reply:
One concern with removing the `_h` is that it's less differentiated from the actual C++ class that it's associated with.  Plus I wonder if people would assume at a glance that it's a struct rather than an opaque handle.